lib/xpm_ruby/job.rb

```diff
@@ -17,5 +17,15 @@ def current(access_token:, xero_tenant_id:)
 
       response["Jobs"]["Job"]
     end
+
+    def add(access_token:, xero_tenant_id:, job:)
+      validated_job = XpmRuby::Schema::Job::Add[job]
```

Comment:
Wonder how we could drop all the `::` operations? Maybe something like:

```
# lib/xpm_ruby/job.rb

require_relative "job/add_schema"

module XpmRuby
  module Job
    def add(...)
      validated_job = AddSchema[job]
    end
  end
end
```

Reply:
Hmm, I will have a think about it. I was following the structure from the client success gem. I think if we keep them in the same relative place in all our gems it might make it easier for our devs to work across gems.

But then again I am not 100% why the client success gem was structured this way. 
;)

Reply:
I suppose one argument for a dedicated schema directory is that theoretically a data structure could be independent of the actions? So xpm_ruby/job.rb is a module containing job related functions, but it is not a class per se. Theoretically any module could access any schema (so one of the calls on client, might use the job schema).

Reply:
I will merge this for now, we can always move it later (pending a discussion).